app/models/order.model.js

```diff
@@ -0,0 +1,72 @@
+const mongoose = require("mongoose");
+
+// Schema
+const Schema = mongoose.Schema;
+const OrderSchema = new Schema({
+  _userId: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "User",
+  },
+  orderItems: [
+    {
```

Comment:
we only need product id, quantity, price, size 
No need for a name and image
add size also if category requires size